tools/c7n_left/c7n_left/policy.py

```diff
@@ -0,0 +1,40 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+#
+import os
+
+from c7n.loader import DirectoryLoader
+from c7n.policy import PolicyCollection, Policy
+from c7n.provider import clouds
+
+
+def load_policies(policy_dir, options):
+    loader = DirectoryLoader(config=options)
+    loader.collection_class = LeftCollection
+    policies = loader.load_directory(policy_dir, recurse=False)
+    if not policies:
+        return ()
+
+    providers = {p.provider_name for p in policies}
+    assert len(providers), "only a single provider per policy dir"
```

Comment:
Message doesn't match the logic. Should this be `assert len(providers) == 1`?

Reply:
thanks, fixed.